app/src/pages/project/ProjectPageQueries.tsx

```diff
@@ -0,0 +1,51 @@
+import { createContext, useContext } from "react";
```

Comment:
This file does feel like it's breaking encapsulation a bit. Though I also can't think of a better solution other than plain not using outlets